### ISSUE TITLE
'Flutter SDK releases' page issue

### ISSUE THREAD

**Issue by Constans** [flutter/website#3983]
Page URL: https://flutter.dev/docs/development/tools/sdk/releases.html
Page source: https://github.com/flutter/website/tree/master/src/docs/development/tools/sdk/releases.md

Found a typo? You can fix it yourself by going to the page source and clicking the pencil icon. Or finish creating this issue.

Description of issue:

On the **Dev channel**, either Windows, Mac, or Linux, versions should start with a **v**. 

There is an issue when switching to a certain version and using the exact version name. For example:
`flutter version 1.18.0-dev.5.0`  will try to switch the version to **1.18.0-de.5.0**
Using `flutter version v1.18.0-dev.5.0` will switch to the correct version **1.18.0-dev.5.0**

I think the cause is that the **flutter version** command strips the first **v** that is found in the version name tag.

**Comment by sfshaza2** [comment]
Yes, this is correct. We have changed the versioning scheme as of 1.17. It is announced on the [Flutter blog](https://medium.com/flutter/flutter-spring-2020-update-f723d898d7af). The [Flutter SDK release page](https://flutter.dev/docs/development/tools/sdk/releases) refers you to the [Flutter wiki](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels), where you can find more info.

Also, I just tried downloading 1.17 and 1.18 from the [release page](https://flutter.dev/docs/development/tools/sdk/releases) and it worked fine.

Are you asking for other doc-related changes?

**Comment by Constans** [comment]
> Yes, this is correct. We have changed the versioning scheme as of 1.17. It is announced on the [Flutter blog](https://medium.com/flutter/flutter-spring-2020-update-f723d898d7af). The [Flutter SDK release page](https://flutter.dev/docs/development/tools/sdk/releases) refers you to the [Flutter wiki](https://github.com/flutter/flutter/wiki/Flutter-build-release-channels), where you can find more info.
> 
> Also, I just tried downloading 1.17 and 1.18 from the [release page](https://flutter.dev/docs/development/tools/sdk/releases) and it worked fine.
> 
> Are you asking for other doc-related changes?

I am talking about the `flutter version version_tag` command not working from a terminal, because of the above-mentioned issue. You can download it via a click.

**Comment by sfshaza2** [comment]
Ooooh! I see. Then you've filed this bug in the wrong place. This bug belongs in the flutter/flutter repo.  :)
